config.json

```diff
@@ -576,6 +576,14 @@
           "sorting"
         ]
       },
+      {
+        "slug": "pop-count",
+        "name": "Pop Count",
+        "uuid": "058c69bd-9c1a-45f0-a94f-62be9f0c9e4e",
+        "practices": [],
```

Comment:
Are there any concepts this exercise practices, or that are required?

Reply:
I wasn't sure. Normally, I'd gravitate towards a `numbers` concept if available for both the practices or requires arrays. Since that's not here, perhaps `basics-2` for both since it touches upon numbers?

Reply:
Yes, I think `basics-2` would be great for the practices fiels, and `basics-1`, basics-2 for the prerequisites. It's nice to have an exercise available so early in the tree.